packages/validator/src/services/block.ts

```diff
@@ -190,10 +196,11 @@ export class BlockProposingService {
   > => {
     // Start calls for building execution and builder blocks
     const blindedBlockPromise = isBuilderEnabled ? this.produceBlindedBlock(slot, randaoReveal, graffiti) : null;
-    const fullBlockPromise = this.produceBlock(slot, randaoReveal, graffiti);
+    const fullBlockPromise =
+      builderSelection !== BuilderSelection.BuilderOnly ? this.produceBlock(slot, randaoReveal, graffiti) : null;
```

Comment:
as mentioned in other comment, should probably just ignore `builderSelection ` if `isBuilderEnabled=false`

Reply:
yes this seems to be the best startegy